pkg/asset/manifests/tectonic.go

```diff
@@ -80,10 +80,17 @@ func (t *Tectonic) Generate(dependencies asset.Parents) error {
 			},
 		}
 	case "openstack":
-		credsEncoded, err := credsFileEncode(openStackCredsFile)
```

Comment:
Looks like you're removing the last consumer here.  Can you drop `credsFileEncode` too?

Reply:
done, thanks a bunch!